core/input/input_event.cpp

```diff
@@ -1549,9 +1557,13 @@ void InputEventAction::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_strength", "strength"), &InputEventAction::set_strength);
 	ClassDB::bind_method(D_METHOD("get_strength"), &InputEventAction::get_strength);
 
+	ClassDB::bind_method(D_METHOD("set_event_id", "id"), &InputEventAction::set_event_id);
+	ClassDB::bind_method(D_METHOD("get_event_id"), &InputEventAction::get_event_id);
+
 	ADD_PROPERTY(PropertyInfo(Variant::STRING_NAME, "action"), "set_action", "get_action");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "pressed"), "set_pressed", "is_pressed");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "strength", PROPERTY_HINT_RANGE, "0,1,0.01"), "set_strength", "get_strength");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "event_id", PROPERTY_HINT_RANGE, vformat("-1,%d,1", int(Input::MAX_EVENT) - 1)), "set_event_id", "get_event_id");
```

Comment:
Missing include for `Input` apparently, but you probably can't include it as that would be a circular dependency, since `input.h` already includes `input_event.h`. Might be easiest to hardcode.